src/applications/disability-benefits/all-claims/validations.js

```diff
@@ -44,3 +44,10 @@ export function validateMilitaryState(errors, state, formData, schema, messages,
     errors.addError('State must be AA, AE, or AP when using a military city');
   }
 }
+
+export const validateBooleanIfEvidence = (errors, fieldData, formData, schema, messages, options, index) => {
```

Comment:
:thinking: So we run the normal validations if we have evidence.

As we're using it in conjunction with USFS' `validateBooleanGroup`, this means we require an evidence type to be selected _if_ we've previously selected `yes` to the `do you have any evidence` question, right?

Took me way too long to figure that one out. :grimacing:

What do you think of `validateIfHasEvidence` or something? (Still not a great name, I know.) This function isn't strictly speaking a boolean validator--that's the `wrappedValidator` we're passing.